src/guide/migration/emits-option.md

```diff
@@ -0,0 +1,62 @@
+---
+title: emits Option
+badges:
+  - new
+---
+
+# `emits` Option <MigrationBadges :badges="$frontmatter.badges" />
+
+## Overview
+
+Vue 3 now offers an `emits` option similar to the existing `props` option. This option can be used to define the events that a component can emit to its parent.
+
+## 2.x Behavior
+
+In Vue 2, you can define the props that a component received, but you can't define which ebents it can emit:
+
+```javascript
+<template>
+  <p>{{ text }}</p>
+  <button v-on:click="$emit('accepted')">OK</button>
+</template>
+<script>
+  export default {
+    props: ['text'],
+  }
+</script>
+```
+
+## 3.x Behavior
+
+Similar to props, the events that the component emit can now be defined with the `emits` option.
+
+```javascript
+<template>
+  <p>{{ text }}</p>
+  <button v-on:click="$emit('accepted')">OK</button>
+</template>
+<script>
+  export default {
+    props: ['text'],
+    emits: ['accepted']
+  }
+</script>
+```
+
+The option also excepts an object notation, which allows to define validators for the arguments that have passed with the emitted event, similar to validators in props definitions.
+
+For more Information on this, please read the [API documentation for this feature](../api/options-data.md#emits).
+
+## Migration Strategy
+
+Even though this is the new optional feature, it is highly recommended that you document all of the emitted events of your components this way because of the [removal of the `.native` modifier](./native-modifier-removed.md).
+
+All events not defined with `emits` are now added as DOM event listeners to the components root node (unless `inheritAttrs: false` has been set).
```

Comment:
I believe *component's* needs an apostrophe.

Reply:
I understand what you mean, but that's not grammatically correct in English, unfortunately.

Reply:
@sdras. This sentence seems to be grammatically correct to me, aside from the missing apostrophe. It does need the reader to parse it the correct way though, so maybe it should be reworded to make it easier to follow?

Reply:
The first paragraph speaks about "components", plural. I'm with Sarah that we can't switch to singular in the next paragraph. But doesn't "components'" exist in English? No sure ^^

Reply:
Trying to make it plural is going to be tricky as ***root node*** is singular. I think ***the components' root nodes*** would be confusing as it could be misconstrued as a reference to fragments.

Personally, I'd try to make the component singular.

Reply:
Did a little change in wording of the previous sentence to make singular work better.

I think it's no at least grammatically correct and gets the point across.